corehq/apps/consumer_user/templates/consumer_user_base.html

```diff
@@ -0,0 +1,67 @@
+{% extends "hqwebapp/base.html" %}
+{% load i18n %}
+
+{% block navigation %}
+<div id="hq-navigation"
+     class="navbar navbar-default navbar-static-top navbar-hq-main-menu">
+  <div class="container-fluid">
+
+    {% if hide_menu %}
+    <nav class="navbar-menus navbar-signin" role="navigation">
+      <div class="nav-settings-bar pull-right">
+        <a href="{% url "consumer_user:consumer_user_login" %}" class="btn btn-primary navbar-btn">{% trans "Sign In"%}
+        </a>
+      </div>
+    </nav>
+
+    {% else %}
+    <nav class="navbar-menus fullmenu collapse" id="hq-full-menu" role="navigation">
+      <div class="nav-settings-bar pull-right">
+        <ul class="nav navbar-nav" role="menu">
+          <li id="settingsmenu-project" class="dropdown">
+            <a class="dropdown-toggle" href="#" data-toggle="dropdown" aria-expanded="true">
+              <span class="responsive-label">{% trans "Menu" %}</span>
+              {% trans "Menu" %}
+              <b class="caret"></b>
+            </a>
+            <ul class="dropdown-menu dropdown-menu-right" role="menu">
+              <li>
+                <a href="{% url "consumer_user:domain_and_cases_list" %}">
+                {% trans "Domains And Cases" %}
```

Comment:
In user-facing content, we refer to domains as "Projects" or "Project Spaces"

Reply:
Made the change